server/game/game.js

```diff
@@ -623,6 +623,35 @@ class Game extends EventEmitter {
         this.continue();
     }
 
+    reInitialisePlayers(swap) {
+        let players = {};
+
+        _.each(this.playersAndSpectators, player => {
```

Comment:
underscore usage is deprecated, please convert this to use a for(foo of bar) loop